google/compute_beta_operation.go

```diff
@@ -0,0 +1,167 @@
+package google
```

Comment:
Not in this PR, but I'd be curious if there's a way to refactor some of the operation stuff so we don't need quite so much duplication between compute_operation and compute_beta_operation. Just something to think about.